docs/as-a-data-transfer-object/collections.md

```diff
@@ -128,7 +129,9 @@ Since the collection type here is a `Collection`, the package will automatically
 
 ## DataCollection's, PaginatedDataCollection's and CursorPaginatedCollection's
 
-The package also provides a few collection classes which can be used to create collections of data objects, it was a requirement to use these classes in the past versions of the package when nesting data objects collections in data objects. This is no longer the case and there are still valid use cases for them.
```

Comment:
Let's keep these on one line, the spate docs site will make sure they're readable

Reply:
Done 😎